src/components/CardContainer/CardContainer.js

```diff
@@ -30,12 +30,28 @@ class CardContainer extends React.Component {
             .then(res => {
                 let random = UniqueRandomArray(res)
                 let dogs = [random(), random()]
-                const state = {...this.state, dogs: dogs}
+                const state = {...this.state, dogs: dogs, randomDog: random}
                 this.setState(state)
                 }
             )
     }
 
+    clickUpdateWin = (id) => {
+        fetch('http://localhost:3000/dogs/' + id + "/win", {
+            method: 'POST'
+        })
+    }
+
+    refreshDogs = () => {
+        let dogs = [this.state.randomDog(), this.state.randomDog()]
```

Comment:
Can we call this `newDogs` or something so we can differentiate it from the actual dogs on the state

Reply:
On it

Reply:
On it